### PR TITLE
LPS-181902 

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPS-181902

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

File: modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro [hunk 1]
```diff
@@ -1019,4 +1156,35 @@ definition {
 		return ${employeeId};
 	}
 
+	macro updateFieldOfEntryById {
+		Variables.assertDefined(parameterList = "${en_US_plural_label},${entryId},${fieldLabel},${fieldValue},${requiredFieldLabel},${requiredFieldValue}");
+
+		var portalURL = JSONCompany.getPortalURL();
+
+		if (${fieldLabel} == "mode") {
```
Comment (yanan) [inline modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro:1164]: Any difference with ${fieldValue} and "${fieldValue}" with if "mode" condition?

**Review by Hijky — APPROVED** [review]

**Comment by magjed4289** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Review by yanan — APPROVED** [review]

**Comment by yanan** [comment]
oops, there are related failed tests on CI

**Comment by magjed4289** [comment]
the test are failing but most probably it's related to some change in the developlment, the tests run OK locally

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**